jcl/src/java.management/share/classes/java/lang/management/LockInfo.java

```diff
@@ -59,13 +54,15 @@ private LockInfo(Object object) {
 	 *             if <code>className</code> is <code>null</code>
 	 */
 	public LockInfo(String className, int identityHashCode) {
-		super();
-		if (className == null) {
-			/*[MSG "K0600", "className cannot be null"]*/
-			throw new NullPointerException(com.ibm.oti.util.Msg.getString("K0600")); //$NON-NLS-1$
-		}
-		this.className = className;
-		this.identityHashCode = identityHashCode;
+		baseInfo = new LockInfoBase(className, identityHashCode);
+	}
+
+	/**
+	 * Constructor for use by subclasses
+	 * @param base base data
+	 */
+	protected LockInfo(LockInfoBase base) {
```

Comment:
A protected constructor cannot be added here, I think removing the protect keyword was missed.

Reply:
You are right: I missed that.